### ISSUE TITLE
fosstodon account add fail

### ISSUE THREAD

**Issue by aliceinwire** [nextcloud/social#438]
```

  | 500  -  {"status":-1,"exception":"OCA\\Social\\Exceptions\\InvalidResourceException","message":"An  exception occurred while executing 'INSERT INTO  `social_cache_documents` (`id`, `type`, `url`, `media_type`,  `mime_type`, `error`, `local_copy`, `public`, `creation`) VALUES(?, ?,  ?, ?, ?, ?, ?, ?, ?)' with params  [\"https:\\\/\\\/cloud.example.com\\\/documents\\\/g\\\/q4p5431e-1t34-1tf4-812c-8738b2839c93\",  \"Image\",  \"https:\\\/\\\/cf.mastohost.com\\\/v1\\\/AUTH_XXXXX\\\/fosstodon\\\/accounts\\\/avatars\\\/000\\\/088\\\/546\\\/original\\\/9ac237b107442c93.jpg\",  \"image\\\/jpeg\", \"\", 0, \"\", \"0\", \"2019-02-24  08:21:52\"]:\n\nSQLSTATE[22001]: String data, right truncated: 1406 Data  too long for column 'url' at row 1"}
-- | --

```

**Comment by aliceinwire** [comment]
resetting didn't help.
I'm on 0.2.0-dev01

also this error:

xt | 500 - {"status":-1,"exception":"OCA\\Social\\Exceptions\\ItemUnknownException","message":""}
-- | --

**Comment by aliceinwire** [comment]
I could solve my issue by getting into mysql and doing a horrible hack :sob: 
```
use nextcloud;
ALTER TABLE social_cache_documents modify url varchar(500);
```

have fun!

**Comment by aliceinwire** [comment]
Probably we could write a migration for it.

**Comment by daita** [comment]
thanks for the catch, i'll work on some migrations !

**Comment by violoncelloCH** [comment]
fixed by #439
